### PR TITLE
Support new Fast-CDR v2.0.0 [19457]

### PR DESCRIPTION
**Pull request by richiware**

### CODE REVIEW

File: fastdds_python/test/types/CMakeLists.txt [hunk 1]
```diff
@@ -74,6 +74,8 @@ include_directories(
     ${PROJECT_SOURCE_DIR}
     )
 
+set_property(SOURCE ${PROJECT_NAME}.i PROPERTY OUTPUT_DIR "${CMAKE_CURRENT_BINARY_DIR}/")
```
Comment (JLBuenoLopez-eProsima) [inline fastdds_python/test/types/CMakeLists.txt:77]: This property is being set but not used later on.

File: fastdds_python_examples/HelloWorldExample/CMakeLists.txt [hunk 2]
```diff
@@ -74,6 +74,8 @@ include_directories(
     ${PROJECT_SOURCE_DIR}
     )
 
+set_property(SOURCE ${PROJECT_NAME}.i PROPERTY OUTPUT_DIR "${CMAKE_CURRENT_BINARY_DIR}/")
```
Comment (JLBuenoLopez-eProsima) [inline fastdds_python_examples/HelloWorldExample/CMakeLists.txt:77]: Same concern as in the previous comment

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]
The types have not been regenerated after the latest changes introduced in Fast DDS-Gen (in plain fix)

File: fastdds_python/test/types/CMakeLists.txt (hunk 1 above)
Comment (richiware) [reply fastdds_python/test/types/CMakeLists.txt:77]: it used by cmake internally

File: fastdds_python_examples/HelloWorldExample/CMakeLists.txt (hunk 2 above)
Comment (richiware) [reply fastdds_python_examples/HelloWorldExample/CMakeLists.txt:77]: it used by cmake internally

**Comment by richiware** [comment]
> The types have not been regenerated after the latest changes introduced in Fast DDS-Gen (in plain fix)

Done in 3595fe1

**Review by JLBuenoLopez-eProsima — DISMISSED** [review]
LGTM. CI will probably fail until eProsima/Fast-DDS#3828 is merged. Labeling as CI pending

**Review by JLBuenoLopez-eProsima — DISMISSED** [review]
LGTM

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM

**Pull request merged**